app/controllers/admin/services_controller.rb

```diff
@@ -16,13 +16,15 @@ def index
     end
 
     def show
-      @latest_metadata = MetadataApiClient::Service.latest_version(params[:id])
-      @service = MetadataPresenter::Service.new(@latest_metadata, editor: true)
+      load_service_and_metadata_corresponding_to_id(params[:id])
       @service_creator = User.find(@service.created_by)
       @version_creator = version_creator
       @published_to_live = published('production')
       @published_to_test = published('dev')
       @versions = MetadataApiClient::Version.all(@service.service_id)
+      if %w[test local].include?(ENV['PLATFORM_ENV'])
```

Comment:
Perhaps is best to use a helper method for this, and use the method in the view? Instead of declaring a variable.
But no biggie.